sapi/cli/php_http_parser.h

```diff
@@ -79,6 +79,9 @@ enum php_http_method
   , PHP_HTTP_POST
   , PHP_HTTP_PUT
   , PHP_HTTP_PATCH
+  /* RFC-2068, section 19.6.1.2 */
```

Comment:
Yes but `RFC-2068, section 19.6.1.2` explains pretty well the 2 methods. There's 8 RFCs about HTTP/1.1, without counting the ones about HTTP/2.0 and so on ;)